mon-pix/tests/unit/components/qrocm-dep-solution-panel_test.js

```diff
@@ -49,6 +53,41 @@ module('Unit | Component | qrocm-dep-solution-panel', function (hooks) {
     });
   });
 
+  module('#getInputClass', function () {
+    test('should return "aband" css class when isAnswerEmpty param is true', function (assert) {
+      //Given
+      const component = createGlimmerComponent('qrocm-dep-solution-panel', { answer: { result: 'ok' } });
```

Comment:
```suggestion
      const component = createGlimmerComponent('qrocm-dep-solution-panel');
```

Supprimer quand inutile